docs/source/platform/federation.mdx

```diff
@@ -0,0 +1,354 @@
+
+## Observability
+
+Like any distributed architecture, you should make sure that your federated GraphQL layer has proper observability, monitoring, and automation to ensure reliability and performance of both your gateway and the federated services underneath it. Serving your GraphQL API from a distributed architecture has many benefits, like productivity, isolation, and being able to match the right services with the right runtimes. Operating a distributed system also has more complexity and points of failure than operating a monolith, and with that complexity comes a need to heighten observability into the state of your system and control over its coordination.
+
+### Enabling Federated Metrics
+
+Apollo Server has support for reporting federated [tracing](/platform/performance/) information from the gateway. In order to support the gateway with detailed timing and error information, implementing services expose their own tracing information per-fetch in their extensions, which are consumed by the gateway and merged together in order to be emitted to the Apollo metrics ingress. To enable this functionality, make sure the `ENGINE_API_KEY` is set in the environment for your gateway server and ensure that all implementing services and the gateway are running `apollo-server` version `2.7.0` or greater. Also, ensure that implementing services do not have the `ENGINE_API_KEY` environment variable set. For more details about how federated tracing works, see [the docs](http://FIXME.com).
+
+Traces will be reported in the shape of the query plan, with each unique fetch to an implementing service reporting timing and error data.
+
+<div style="text-align:center">
+    <img src="../images/federated_trace.png" alt="Federated Trace">
+    </img>
+</div>
+
+Operation-level statistics will still be collected over the operations sent by the client, and those operations will be validated as part of the `service:check` validation workflow.
+
+### Inspecting Query Plans
+
+<!--
+add links to advanced-features for
+- [`@requires`](/federation/advanced-features/#computed-fields)`
+-`[`@provides`](/federation/advanced-features/#using-denormalized-data)`
+ -->
+
+When the gateway receives a new query, it generates a query plan that defines the sequence of requests the gateway will send to the necessary downstream services. Inspecting a query plan can be a helpful tool in understanding the gateway and exploring how directives like `@requires` and `@provides` can help optimize query plans. To make it easy to access query plans, the `@apollo/gateway` package includes a build of GraphQL Playground that adds a query plan inspector.
```

Comment:
Updated to say that AS includes our version of playground rather than the gateway package

```suggestion
When the gateway receives a new query, it generates a query plan that defines the sequence of requests the gateway will send to the necessary downstream services. Inspecting a query plan can be a helpful tool in understanding the gateway and exploring how directives like `@requires` and `@provides` can help optimize query plans. To make it easy to access query plans, Apollo Server includes a build of GraphQL Playground that adds a query plan inspector.
```